modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/ModelValidationUtils.java

```diff
@@ -63,4 +65,13 @@ static void validateConfigParametersNamesNotAllowed(ParameterizedModel model, Pr
                                                 NAME_ATTRIBUTE_NAME))));
   }
 
+  static boolean overrideEqualsAndHashCode(Class<?> clazz) {
```

Comment:
This method will probably only be used for the validation introduced for this issue. If we create a new validator I think it is better to put it there instead of this utils class.